### PR TITLE
Test cleanup 0910

### PR DESCRIPTION
**Pull request by jthelin**
TestInput directory is no longer used / required.

 - Remove reference to TestInput directory from test settings because it is no longer used / required.

Fix some minor build warnings in test cases.

- Use of grainId local variable in these methods was hiding field in class, causing build warnings.

### CODE REVIEW

File: src/Tester/GenericGrainTests.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ MIT License
 using UnitTests.Tester;
 using System.Collections.Generic;
 
-namespace UnitTests.General
```
Comment (gabikliot) [inline src/Tester/GenericGrainTests.cs]: all our namespaces in this project are: `namespace UnitTests.General`. Don't see a reason to change everything now.

File: src/Local.testsettings [hunk 2]
```diff
@@ -1,10 +1,6 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <TestSettings name="Local" id="8a37c363-3fe3-4ab6-a076-e4fcf4d95a30" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
   <Description>These are default test settings for a local test run.</Description>
-  <Deployment>
```
Comment (sergeybykov) [inline src/Local.testsettings]: I wonder if this may break our internal tests that we are running via VSO.

**Comment by sergeybykov** [comment]
This one seems to have caused massive failures of functional tests in VSO.

**Comment by jthelin** [comment]
Withdrawing this change until the internal functional tests can be migrated / fixed.

**Pull request closed without merging**